src/wazuh_modules/vulnerability_scanner/tests/component/vulnerabilityScannerFacade_test.cpp

```diff
@@ -0,0 +1,60 @@
+/*
+ * Wazuh Vulnerability Scanner - Unit Tests
+ * Copyright (C) 2015, Wazuh Inc.
+ * September 21, 2023.
+ *
+ * This program is free software; you can redistribute it
+ * and/or modify it under the terms of the GNU General Public
+ * License (version 2) as published by the FSF - Free Software
+ * Foundation.
+ */
+
+#include "vulnerabilityScannerFacade_test.hpp"
+#include "vulnerabilityScannerFacade.hpp"
+#include <external/nlohmann/json.hpp>
+#include <memory>
+
+/*
+ * @brief Log function used by VulnerabilityScannerFacade.
+ */
+// LCOV_EXCL_START
+void logFunction(const modules_log_level_t logLevel, const std::string& logMessage)
+{
+    std::ignore = logLevel;
+    std::ignore = logMessage;
+}
+// LCOV_EXCL_STOP
```

Comment:
NOTE: The tests files shouldn't be tested for coverage, there something wrong on the RTR

Reply:
Addressed here:
- #19309